### ISSUE TITLE
Missing unit test and functional test in csi module

### ISSUE THREAD

**Issue by leonwanghui** [opensds/nbp#214]
**Is this a BUG REPORT or FEATURE REQUEST?**:

/kind feature

**What happened**:
I found there is a [csi-test](https://github.com/kubernetes-csi/csi-test) project located in `k8s-csi` code repo, I think it's required to import that project into csi module and trigger that in ci environment.

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:
@xing-yang Please take a look at that project.

**Environment**:
- NBP version:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by xing-yang** [comment]
@leonwanghui Yes, it is good to use csi-test to test CSI plugin.  We should add this task.

**Comment by xing-yang** [comment]
@wisererik

**Comment by xing-yang** [comment]
Here are instructions on how to use the sanity test package to test CSI drivers: https://github.com/kubernetes-csi/csi-test/tree/master/pkg/sanity

**Comment by wisererik** [comment]
Sure, I saw that, @jackhaibo  please add test for CSI

**Comment by skdwriting** [comment]
@wisererik @jackhaibo any update on this?
